pkg/format/format.go

```diff
@@ -0,0 +1,36 @@
+package format
```

Comment:
NIT: If we are going to add a package `format` I'd rename the files to know what they are formatting, for example this one could be `k8s.go`. 

I don't know if there will be lots of formatting functions, Should we create a package then? I'm thinking out loud to get some more opinions

Reply:
> NIT: If we are going to add a package `format` I'd rename the files to know what they are formatting, for example this one could be `k8s.go`.
>
> I don't know if there will be lots of formatting functions, Should we create a package then? I'm thinking out loud to get some more opinions

Maybe changing the file to `k8s.go` could be done, but maybe this transformations can apply to other things. As you wish, but maybe we can leave it as it is now and see how it evolves